www/settings/custom_metrics/imgs-too-large.js

```diff
@@ -0,0 +1,34 @@
+var imgsTooLarge = [];
+var allImgs = document.querySelectorAll("img");
+var widthTolerance = 0.5;
+for (var i = 0; i < allImgs.length; i++) {
+  var boundingClientRect = allImgs[i].getBoundingClientRect();
+  //getBoundingClientRect returns 0 if the image is hidden (display none or parent display none)
+  //display none isn't enough (only catches if that image is display non, not parent)
+  //so we check offsetParent, but also position fixed (in everyone but ff, position fix makes offsetParent null)
+  let isVisible =
+    window.getComputedStyle(allImgs[i]).display != "none" &&
+    (allImgs[i].offsetParent != null ||
+      window.getComputedStyle(allImgs[i]).position == "fixed");
+  // if display width is less than 50% of the full image natural size
+  if (
+    isVisible &&
+    boundingClientRect.width < allImgs[i].naturalWidth * widthTolerance
```

Comment:
This is going to catch svgs too, right? Where `naturalWidth` is way higher but that's probably ok.

Probably not worth special-casing SVG since the goal is to experiment